api/pending.ts

```diff
@@ -1,33 +1,40 @@
-import { NowRequest, NowResponse } from '@now/node';
-import Web3 from 'web3'
-import BigNumber from 'bignumber.js'
+import { NowRequest, NowResponse } from "@vercel/node";
+import Web3 from "web3";
+import BigNumber from "bignumber.js";
 
-const chefABI = require('../contracts/chef')
+const chefABI = require("../contracts/chef");
 
 const getBalanceNumber = (balance: any, decimals = 18) => {
-  const displayBalance = balance.dividedBy(new BigNumber(10).pow(decimals))
-  return displayBalance.toNumber()
-}
+  const displayBalance = balance.dividedBy(new BigNumber(10).pow(decimals));
+  return displayBalance.toNumber();
+};
 
 const web3 = new Web3(
-    new Web3.providers.HttpProvider(
-        "https://bsc-dataseed.binance.org"
-    )
+  new Web3.providers.HttpProvider("https://bsc-dataseed.binance.org")
```

Comment:
Is it possible to use the Web3 Provider that is exported from the `lib`?

Reply:
Fixed with https://github.com/pancakeswap/gatsby-pancake-api/pull/3/commits/6fd8b808b5277a07867d869df54ddec098c8e616